lib/consumer.ts

```diff
@@ -1,17 +1,18 @@
 import { CoreKernel } from '@balena/jellyfish-core';
 import { defaultEnvironment } from '@balena/jellyfish-environment';
-import { getLogger, LogContext } from '@balena/jellyfish-logger';
-import * as metrics from '@balena/jellyfish-metrics';
-import {
+import { getLogger } from '@balena/jellyfish-logger';
+import type { LogContext } from '@balena/jellyfish-logger';
+import { markJobAdd, markJobDone } from '@balena/jellyfish-metrics';
```

Comment:
I'm assuming you've seen some of my commits where I did something similar (e.g. for the queue). In those cases it's useful because of the `import type`, and IMO importing types directly is most of the times simpler. But I personally don't directly import functions because they usually have names that are way too general. So something like `core.create` tends to be much more readable than just `create`.

I avoided changing things to fit my style and was mostly thinking about the `import type` thing. But now that I think about it, you can probably `import type * as ...` :p

Reply:
Sure, I was debating it myself. I have no problems going with `core.create` etc if it matches the rest of the code base.